src/app/Models/Order.php

```diff
@@ -17,63 +22,70 @@ class Order extends Model
      * $this->attributes['updated_at'] -timestamp -contains the order update date
      */
```

Comment:
Primary key doesn't belong to product

Reply:
I guess this is something that is debatable but should we put foreign key or just User because that is the data type we are going to return